package.json

```diff
@@ -23,9 +23,9 @@
     "lock": "npm run lint:versions -s && npm shrinkwrap --dev && sed -i '' 's/https:/http:/g' npm-shrinkwrap.json"
   },
   "dependencies": {
-    "jquery": "^2.2.4",
-    "uikit": "3.0.0-beta.23",
-    "throttle-debounce": "^1.0.1"
+    "jquery": "2.2.4",
+    "uikit": "3.0.0-beta.27",
+    "throttle-debounce": "1.0.1"
```

Comment:
There are a lot of new issues introduced in `npm@5` that haven't been resolved yet. (https://github.com/npm/npm/issues?q=is%3Aopen+is%3Aissue+label%3Anpm5)

For me, the hassle of continually updating all the Dockerfiles to the next npm5 dot release is too much (and the risk of not locking it down to a minor release is also too much).

I want to be on the npm5 train for your reasons but I am also hesitant to introduce new software so quickly now.

I haven't looked into any pull request bots but I've seen a few around, maybe we can find one that's free.